lnbits/static/css/base.css

```diff
@@ -528,7 +528,9 @@ video {
 
 .q-card.q-card--dark,
 .q-date--dark {
-  box-shadow: none;
+  border: 1px solid rgba(0,0,0,.12);
```

Comment:
@arcbtc i think you need to add this to `base.scss` the base.css will be overwritten by `make bundle`